sites/default/documents/holidays_storage/readme.txt

```diff
@@ -0,0 +1 @@
+/*Use this folder to add custom files to be used by the system*/
```

Comment:
Also, no need for this file or creating this directory. Better to just create the directory if it doesn't yet exist when upload the csv file.